crates/matrix-sdk-ui/src/spaces/room_list.rs

```diff
@@ -263,27 +266,44 @@ impl SpaceRoomList {
                     result.rooms.into_iter().partition(|f| f.summary.room_id == self.space_id);
 
                 if let Some(room) = space.first() {
-                    let mut space = self.space.write();
+                    let mut via_parameters = self.via_parameters.lock();
+                    *via_parameters = Some(
+                        room.children_state
+                            .iter()
+                            .filter_map(|event| match event.deserialize() {
+                                Ok(child) => Some((child.state_key, child.content.via)),
+                                Err(error) => {
+                                    error!("Failed deserializing space child event: {error}");
```

Comment:
Probably just a `warn!()`? Since we're just skipping things.

Reply:
Fair 👍